packages/wonder-blocks-color/src/index.ts

```diff
@@ -48,12 +51,16 @@ const Color: ColorType = {
     white,
     white64: fade(white, 0.64),
     white50: fade(white, 0.5),
+    white32: fade(white, 0.32),
```

Comment:
**suggestion:** could you please move this to `tokens.ts`? The mid-term plan is to get rid of this `color` package and instead rely directly on the `theming` package for colors and spacing (and any other tokens).

https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-theming/src/tokens.ts#L15